airbyte-integrations/connectors/source-lever-hiring/source_lever_hiring/spec.json

```diff
@@ -29,12 +29,12 @@
               "client_id": {
                 "title": "Client ID",
                 "type": "string",
-                "description": "The Client ID of your application"
+                "description": "The Client ID of your Lever Hiring developer application"
               },
               "client_secret": {
                 "title": "Client Secret",
                 "type": "string",
-                "description": "The client secret of your application",
+                "description": "The client secret of your Lever Hiring application",
```

Comment:
"description": "The client secret of your Lever Hiring developer application",

missed "developer"